clients/src/main/java/org/apache/kafka/common/security/authenticator/AbstractLogin.java

```diff
@@ -58,14 +60,14 @@ public LoginContext login() throws LoginException {
         if (jaasConfigFile == null) {
             log.debug("System property '" + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' is not set, using default JAAS configuration.");
         }
```

Comment:
Have updated the message, moved it to `JaasUtils` where configuration is loaded. Rest of the code can assume that the config is valid.